frontend/app/app.js

```diff
@@ -1,8 +1,8 @@
 import Application from '@ember/application';
```

Comment:
I'm on commit 9d6ec5764af8df18dc95062b44a2a062e8762b2e

I'm going to jsut trust this is all automated stuff